src/en/infernalvoidscans/build.gradle

```diff
@@ -2,8 +2,9 @@ ext {
     extName = 'Infernal Void Scans'
     extClass = '.InfernalVoidScans'
     themePkg = 'mangathemesia'
-    baseUrl = 'https://hivetoon.net'
-    overrideVersionCode = 9
+    baseUrl = 'https://hivetoon.com'
+    overrideVersionCode = 10
+    isNsfw = true
```

Comment:
What series specifically makes it NSFW?

Reply:
> What series specifically makes it NSFW?

- [The Best Noble in Another World: The Bigger My Harem Gets, the Stronger I Become](https://hivetoon.com/manga/the-best-noble-in-another-world-the-bigger-my-harem-gets-the-stronger-i-become/) - Chapter 2
- [Brutal: Confessions of a Homicide Investigator](https://hivetoon.com/manga/brutal-confessions-of-a-homicide-investigator/) - Chapter 3

```Adult``` option available in the filters

Reply:
but generally looks safe. not sure since we got no thing in between safe and nsfw

Reply:
I agree, it isn't really nsfw as a whole